src/pages/Staking/HowItWorks/StakingApplications/index.tsx

```diff
@@ -109,7 +115,8 @@ const StakingApplications: PageComponent = () => {
                 <Stack>
                   <BodyMd>Change your authorized amount at any time. </BodyMd>
                   <BodyXs>
-                    There is a deauthorization cooldown period of 45 days.
+                    There is a deauthorization cooldown period of 45 days for
+                    tBTC and Random Beacon apps and 180 days for TACo.
```

Comment:
Should we be consistent with what we have in other places i.e. 6 months. Technically it is 182 days I believe.
```suggestion
                    tBTC and Random Beacon apps and 6 months for TACo.
```